ticket.go

```diff
@@ -308,30 +308,27 @@ func (s *TicketService) Create(ticket *Ticket) (*Ticket, *Response, error) {
 }
 
 // Update a Zendesk Ticket
-func (s *TicketService) Update(ticket *Ticket) (*Ticket, *Response, error) {
+func (s *TicketService) Update(ticket *Ticket) error {
```

Comment:
Let's not get rid of the ticket and the response if it's not necessary. Is it necessary?